jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantRenderer/components/util.ts

```diff
@@ -0,0 +1,133 @@
+import { readConfObject } from '@jbrowse/core/configuration'
+import SceneGraph from '@jbrowse/core/util/layouts/SceneGraph'
+import { Feature } from '@jbrowse/core/util/simpleFeature'
+import { AnyConfigurationModel } from '@jbrowse/core/configuration/configurationSchema'
+import Box from './Box'
+import Chevron from './Chevron'
+import Diamond from './Diamond'         // new
+import ProcessedTranscript from './ProcessedTranscript'
+import Segments from './Segments'
+import Subfeatures from './Subfeatures'
+// TODO - better extend SvgFeatureRenderer plugin
+
+interface Glyph extends React.FunctionComponent {
+  layOut?: Function
+}
+
+export function chooseGlyphComponent(feature: Feature): Glyph {
+  const type = feature.get('type')
+  const strand = feature.get('strand')
+  const subfeatures: Feature[] = feature.get('subfeatures')
+
+  if (subfeatures) {
+    const hasSubSub = subfeatures.find(subfeature => {
+      return !!subfeature.get('subfeatures')
+    })
+    if (hasSubSub) {
+      return Subfeatures
+    }
+    const transcriptTypes = ['mRNA', 'transcript']
+    if (
+      transcriptTypes.includes(type) &&
+      subfeatures.find(f => f.get('type') === 'CDS')
+    ) {
+      return ProcessedTranscript
+    }
+    return Segments
+  }
+  if(type === "SNV"){       // new for diamond implementation
```

Comment:
can you make these comments clearer the search and find? Either a "//TODO: added for XXX, should remove", or put some string in them to tag them and make it easier to globally search and find them across the project?

Reply:
Updated. Search term is "NEW"